gapii/cc/call_observer.cpp

```diff
@@ -132,6 +132,21 @@ void CallObserver::encode(const ::google::protobuf::Message* cmd) {
   encoder()->object(cmd);
 }
 
+void CallObserver::reenter() {
```

Comment:
The name of this function made me think we were doing something with reentrancy in the observer. Sugguest changing to resumeObservation, finishObservation, or something similar.

Reply:
Good point, I renamed to `resume()` to stay homogeneous with other methods names of this class (enter(), exit()).